doc/Bugsee.md

```diff
@@ -0,0 +1,141 @@
+# Bugsee
+
+This project include [Bugsee](https://www.bugsee.com/) reporting and Logging, for both Android and iOS apps.
+Bugsee lets you monitor and get instant log of unhandled exceptions with traces, events, stacktrace and videos/screenshots of the reported exception. More features are provided by Bugsee like data obscuration and log filter.
+
+For this implementation we've used [bugsee_flutter](https://pub.dev/packages/bugsee_flutter) package.
+
+- By default only release apps will have Bugsee reporting enabled, to enable Bugsee in debug mode add your token in `launch.json` add remove the check on debug mode in `BugseeManager` class.
+- **Token**
+    - Generate your token in order to test Bugsee logging and reporting 
+        - Head to [Bugsee dashboard](https://www.bugsee.com/)
+        - Create a new account
+        - Create a new Android/iOS (choose Flutter framework)
+        - Copy-paste the generated token into `launch.json`
+
+
+## 1) Features
+
+In this project we've implemented the following features of Bugsee:
+- [Manual invocation](https://docs.bugsee.com/sdk/flutter/manual/) helpfull for developers to test their Bugsee integration and setup with new tokens.
+- [Custom data](https://docs.bugsee.com/sdk/flutter/custom/) custom data could be attached to the logged exceptions (emails or other type of data)
+    - [Email](https://docs.bugsee.com/sdk/flutter/custom/#:~:text=Adding%20custom%20data-,User%20email,-When%20you%20already) this will identify the user whom experienced the exception/bug this will update the exception dashboard item from anonymous to the user's mail this data will be reported with every logged exception unless the app is deleted or removed manually.
```

Comment:
Make sure you use proper English sentences with correct grammar and punctuation. Here it feels like it should be 2 sentences.
> (...) users:s mail. This data will (...)